src/ui/interface.js

```diff
@@ -658,6 +665,32 @@ export class UI {
 		}
 	}
 
+	
+	showBonusPotential(){
+		//Shows bonus capability 
+
+		const game = this.game;
+		this.abilitiesButtons.forEach((btn) => { 
+			const ability = game.activeCreature.abilities[btn.abilityId];
+
+			//The executioner Axes for Golden Wyrm jumps to the right
```

Comment:
This should be more generic, reusable for other abilities. Meaning no mention of specific unit and ability in `interface.js`.